src/js/experiments/languageGuides.js

```diff
@@ -0,0 +1,42 @@
+import { reconstructToC, highlightTocOnScrollOnce } from '../site/toc';
+import { displayBlockElement, displayInitialElement } from '../utils';
+
+// https://app.optimizely.com/v2/projects/16812830475/experiments/20872380274/variations
+export default () =>
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_add_video_tab_to_docs_test',
+    groupExperimentName: 'q4_fy22_docs_disco_experiment_group_test',
+    experimentContainer: '.treatment',
```

Comment:
This could be potentially dangerous to have a container called just `treatment`. Can we make it more specific to that experiment, please?